src/util/http.ts

```diff
@@ -25,6 +26,9 @@ export async function apiAvailable(): Promise<boolean> {
 export async function getRequest<ResponseType>(url: string): Promise<ResponseType> {
     if (!url.startsWith("/")) url = "/" + url;
     const rawResponse = await fetch(process.env.VUE_APP_API_URL + url);
+    if (!rawResponse.ok) {
+        throw new Error(rawResponse.statusText);
```

Comment:
If you are throwing an error here you also need to catch it wherever getRequest is used. e.g. IDENTITY_PALLET_EXISTS. Please review